### ISSUE TITLE
Main text on Cards hover

### ISSUE THREAD

**Issue by successbyte** [ehsanullahhaidary/dribble-clone#12]
# New Issue

The cards at middle section has a text when hover the mouse on the cards and the text has no hover functionality I want when the user hover mouse on it the mouse cursor to be pointer 

## Actual behavior
when hover mouse on the text the mouse cursor does not change

![Screenshot 2024-09-10 220850](https://github.com/user-attachments/assets/e1ae2858-b261-44ec-9daa-be600c5b3d3c)

## Expected behavior

I want the mouse to change to `cursor: pointer`

**Comment by successbyte** [comment]
if you have any question regarding the issue or more info on Open Source ask here or reach out to me
https://x.com/Ehsan_Haidary

**Comment by Rahul-2006** [comment]
HI @ehsanullahhaidary please assign this to me
i can do it...

**Comment by ehsanullahhaidary** [comment]
Hi @Rahul-2006 
I assigned you the #10 
this repo is for first timers only after working on above issue you can search for some more difficult issues and contribute
or if you have friend want to contribute for first time then welcome 
good luck bro

**Comment by ehsanullahhaidary** [comment]
Great @Rahul-2006 thanks

**Comment by aasthajajoo123** [comment]
@successbyte hi! can you assign me this issue?

**Comment by successbyte** [comment]
Hi @aasthajajoo123 assigned you 
you can make a PR
please give the repo a star and tell your friends about the repo if they want to contribute for the first time
thanks

**Comment by successbyte** [comment]
Hi @aasthajajoo123  what did you do with the issue?
